_includes/mobile/rewarded-server-side-configuration.md

```diff
@@ -0,0 +1,68 @@
+##### Server-side Rewarded Ad Unit Configuration
+
+The Rewarded Ad Unit assumes special behavior that should be configurable by the platform or publisher according to the application or ad experience guides.  
+
+Configuration of rewarded ad unit can be done using stored impression-level stored request and the [passthrough](https://docs.prebid.org/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#request-passthrough) feature of Prebid Server. 
```

Comment:
Sorry, not following this and the example below. The PBS `passthrough` feature is supported at both the global and the impression-level.

The example below shows a global passthrough, but the text here implies impression-level.

Also, I think the wording here would be clearer if it said "... by defining the Prebid Server [passthrough extension] from the app or using an stored impression-level stored request on the server."

Reply:
Hi Bret! Thanks for catching it! 

Yes, the proposal is to use the impression-level passthrough. We clarified the expected place in the response.